common/metrics/metricstest/metricstest_test.go

```diff
@@ -68,3 +69,40 @@ func TestBasic(t *testing.T) {
 	require.Equal(t, float64(2), s2.MustCounter(counterName+"_total", expectedCounterTags...))
 	require.Equal(t, float64(10), s2.MustGauge(gaugeName, expectedGaugeTags...))
 }
+
+func TestHistogram(t *testing.T) {
+	logger := log.NewTestLogger()
+	handler := MustNewHandler(logger, metrics.ClientConfig{
```

Comment:
Please don't allow panics in tests since it will crash the whole test suite, skipping all subsequent tests. Prefer this alternative which will only fail the current test.

```go
h, err := NewHandler(...)
require.NoError(t, err)
```

Reply:
good idea, went ahead and refactored out all the `Must...` functions, and checked for an error after each was called